src/ui/categories/common/CategoryExpandedIcon.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
This is basically a copy of PageExpandedIcon, why not moving PageExpandedIcon to common directory, renaming it in "TreeNodeExpandedIcon" and reuse it in both places?